labgrid/util/agents/sysfsgpio.py

```diff
@@ -2,6 +2,7 @@
 This module implements switching GPIOs via sysfs GPIO kernel interface.
 
 Takes an integer property 'index' which refers to the already exported GPIO device.
+Takes a boolean property 'invert' which inverts logical values if set to True (active-low)
```

Comment:
Why do we have to pass this down to the agent? Isn't it enough to do the invert handling in the driver as for, i.e. `ModbusCoilDriver`?

Reply:
As the export of the gpio line is happening in the ```__init__``` method and setting active_low happs directly afterwards, then knowing the value of invert is needed (see lines 44-46)

Reply:
But we don't have to use the `active_low` property in the sysfs if we invert in the driver or am I missing something?
`set()/get()` can perform the invert without touching the other sysfs property.

Reply:
I am not sure whether changing the start params is not a breaking change in the coordinator <-> exporter interface.

Reply:
Unless there is a way to explicitly test your concern, then I don't really know what to say.  

But I can say that creating a SysfsGpio resource either with the invert flag or without out works fine.  Acquiring those resources works, having the resources as named resources also works.  Accessing the resource either locally or remotely works.

What kind of conditions are you thinking of which are different from the ones described above?

Reply:
Is there some way to test the coordinarot<->exporter interface to see if anything is broken?

Reply:
> I am not sure whether changing the start params is not a breaking change in the coordinator <-> exporter interface.

Since version 25.0 will not be compatible with earlier version, would it make sense to incorporate these changes before the release?